pkg/invariants/network/disruptionpodnetwork/invariant.go

```diff
@@ -153,8 +168,12 @@ func (pna *podNetworkAvalibility) CollectData(ctx context.Context, storageDir st
 			}
 		}
 	}
+	junits = append(junits, &junitapi.JUnitTestCase{
+		Name:      "[sig-network] poller pod logs",
```

Comment:
Test name looks a little generic, and a good thing to get right the first time it goes in.

"can collect pod-to-pod poller pod logs"?

Reply:
Shouldn't this test fail if you hit an error?

Reply:
I know you prefer to just merge an then sort these out, but technically it would be nice to keep this test out of all the databases if it's short lived, suggest removing the junit if you want to instamerge and then bring it back when it's named in final form.